js/blocks/GraphicsBlocks.js

```diff
@@ -828,7 +856,7 @@ function setupGraphicsBlocks(activity) {
         flow(args, logo, turtle, blk) {
             if (args[1] === undefined) return;
             if (args[0] === null) activity.errorMsg(NOINPUTERRORMSG, blk);
-
+            
```

Comment:
Please remove the extra spaces added to Line 859

Reply:
Otherwise, it looks good to go :+1:

Reply:
@walterbender, I just removed it, my apologies for the oversight.